src/main/java/com/salesforce/mirus/offsets/MirusOffsetTool.java

```diff
@@ -97,7 +101,14 @@ private void run() throws IOException {
     if (args.describe) {
       offsetFetcher.start();
       try {
-        offsetSerDe.write(offsetFetcher.readOffsets(), System.out);
+        Stream<OffsetInfo> offsetInfos =
+            offsetFetcher
+                .readOffsets()
+                .filter(
+                    offsetInfo ->
+                        offsetInfo.offset != null
```

Comment:
I think this clause just reduces to `offsetInfo.offset != null || args.writeNullOffsets`